cli/src/main/java/com/devonfw/tools/ide/tool/mvn/Mvn.java

```diff
@@ -96,12 +97,10 @@ private void createSettingsSecurityFile(Path settingsSecurityFile) {
       secureRandom.nextBytes(randomBytes);
       String base64String = Base64.getEncoder().encodeToString(randomBytes);
 
-      ProcessContext pc = this.context.newProcess().executable("mvn");
-      pc.addArgs("--encrypt-master-password", base64String);
-
-      ProcessResult result = pc.run(ProcessMode.DEFAULT_CAPTURE);
-
-      String encryptedMasterPassword = result.getOut().get(0);
+      Mvn mvn = new Mvn(context);
+      ProcessResult result = mvn.runTool(ProcessMode.DEFAULT_CAPTURE, ProcessErrorHandling.LOG_WARNING, this.context.newProcess(), "--encrypt-master-password",
+          base64String);
+      String encryptedMasterPassword = result.getOut().getFirst();
```

Comment:
Just as nice to have:
We have `ProcessContext.runAndGetSingleOutput`. The logic from this method could be extracted to `ProcessResult` for reuse in places like this.
https://github.com/devonfw/IDEasy/blob/9e6319a3fcb7c6ae45b77f5f3056b9cfd2ba8cde/cli/src/main/java/com/devonfw/tools/ide/process/ProcessContext.java#L174-L202

Reply:
I've moved this code to `ProcessResult` now.